python/eos/tasks.py

```diff
@@ -73,16 +77,17 @@ def task_wrapper(*args, **kwargs):
                     display(iaccordion)
                 # use invocation-specific ipython output widget (if available)
                 with ioutput:
-                    result = func(*args, **kwargs)
+                    result = func(**_args)
```

Comment:
Why did you have to remove `**kwargs` here?

Reply:
I did not. Actually, everything is a keyword argument now. I have to modify some arguments by name earlier in the code, and assemble a dictionary of all of the tasks arguments. Nothing is provided using positional arguments anymore.